src/Session.java

```diff
@@ -1,72 +1,128 @@
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
 import java.util.List;
 import java.util.Scanner;
 
 public class Session {
-    private static final int MAX_LIVES = 7;
 
-    private List<Player> players;
-    private int lives;
-    private Word word;
-    private Player leader;
-    private Scanner input;
+  private static final int MAX_LIVES = 7;
 
-    public Session(Scanner input, List<Player> players, Player leader, Word word, int lives) {
-        this.lives = lives;
-        this.players = players;
-        this.leader = leader;
-        this.word = word;
-        this.input = input;
-    }
+  private String[] indexLeaderBoard = new String[]{
```

Comment:
indexLeaderBoard? That tells me absolutely nothing about what this array is for.

To me, this looks more of a "livesLeftToHangmanImage"